include/isdimm_vpd_parser.hpp

```diff
@@ -0,0 +1,135 @@
+#pragma once
+
+#include "ipz_parser.hpp"
+#include "parser_interface.hpp"
+#include "types.hpp"
+
+namespace vpd
+{
+using ParserInterface = vpd::ParserInterface;
+using kwdVpdMap = types::KeywordVpdMap;
+
+class isdimmVpdParser : public ParserInterface
```

Comment:
class name should be starts with caps , as upper case letters as word separators, to keep consistency of style across the repo.

Reply:
Acknowledged.